scripts/load-images.ps1

```diff
@@ -0,0 +1,6 @@
+# Load CAPI
+# FIXME(sbueringer)
```

Comment:
Hm. I think we shouldn't need this if we use preload images when we create the self-hosted cluster